mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml

```diff
@@ -214,4 +210,45 @@
      When the described button is pressed, the overlay will open and the browser will be hidden underneath it -->
     <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for %1$s Home, search, settings, and more</string>
 
+    <!-- Firefox Accounts Strings-->
+    <!-- Description for Sent tab buttons (in navigation toolbar & settings, both on the homescreen)
+     that will launch the Firefox Account sign-in experience so users can receive Firefox tabs on their Fire TV.
+     %1$s will be replaced with Firefox. -->
+    <string name="fxa_navigation_item_new">New! Send %1$s tabs to Fire TV</string>
+    <!-- Description for Sent tab button in navigation toolbar
+     that will launch the Firefox Account settings after the user is signed in. -->
+    <string name="fxa_navigation_item_signed_in">Sent tab settings</string>
```

Comment:
I wonder if this is `tabs` or `tab`, i.e., most other strings use plural.

Reply:
Yes, that's a good point. Where do the changes get made? Do I re-land this string in fftv repo?

Reply:
Yes. And with the process we have here with strings in quarantine, you won't need to change the ID. ID changes become necessary once the quarantine PR is merged to master here, and we actually expose the strings to localizers.

Reply:
I don't think this change was brought into this PR, afaict. Can it still be exported before the strings are exposed to localizers so we can keep the id the same?

Reply:
Sorry, I merged thinking they had all been taken care of already.
We'll need an ID change now :/